### PR TITLE
Use dvs' uuid as a prefix, update dvportgroup names on agent boot

### PR DESCRIPTION
**Pull request by spaskalev**

### CODE REVIEW

File: networking_dvs/utils/dvs_util.py [hunk 1]
```diff
@@ -445,16 +436,35 @@ def create_dvportgroup(self, sg_attr_key, sg_set, port_config):
...
             raise exceptions.wrap_wmvare_vim_exception(e)
 
+    def dvportgroup_name(self, sg_set):
+        """
+        Returns a dvportgroup name for the particular security group set
+        in the context of the current switch
+        """
+        # There is an upper limit on managed object names in vCenter
+        dvs_id = ''.join(self.uuid.split(' '))[:8]
+        name = dvs_id + "-" + sg_set
```
Comment (fwiesel) [inline networking_dvs/utils/dvs_util.py]: Could you change it back to 

`name = sg_set + "-" + dvs_id`

after all? I was first thinking, it was better to order it first by switch, then by security group, but then you can do that by simply going on the switch and listing the port-groups there.

**Review by fwiesel — CHANGES_REQUESTED** [review]

File: networking_dvs/utils/dvs_util.py [hunk 2]
```diff
@@ -445,6 +436,23 @@ def create_dvportgroup(self, sg_attr_key, sg_set, port_config):
...
+        if len(name) > 80:
+            # so we use a hash of the security group set
+            hex = hashlib.sha224()
+            hex.update(dvs_id)
```
Comment (fwiesel) [inline networking_dvs/utils/dvs_util.py]: And maybe remove the `dvs_id` from the hash.
We have the suffix already in the name, and then the same set of security groups hashes to the same value. Which is a plus, in my mind.

File: networking_dvs/utils/dvs_util.py (hunk 1 above)
Comment (spaskalev) [reply networking_dvs/utils/dvs_util.py]: Done

File: networking_dvs/utils/dvs_util.py (hunk 2 above)
Comment (spaskalev) [reply networking_dvs/utils/dvs_util.py]: True that, done

**Review by fwiesel — APPROVED** [review]

**Pull request merged**